apps/address-service/domains/common/utils/services/suggest/SuggestionProviderDetector.js

```diff
@@ -0,0 +1,29 @@
+const {
+    DadataSuggestionProvider,
+    GoogleSuggestionProvider,
+} = require('@address-service/domains/common/utils/services/suggest/providers')
+
+class SuggestionProviderDetector {
+
+    /**
+     * @param {string} geo
+     * @returns {AbstractSuggestionProvider}
+     */
+    getProvider (geo) {
+        /** @type {AbstractSuggestionProvider} */
+        let suggestionProvider
+        switch (geo) {
+            case 'dadata':
```

Comment:
looks strange %)

Reply:
It will be helpful to other services which using dadata now. These services won't be forced to change internal logic immediately.

Reply:
Could you please comment the code about this logic

Reply:
Done. The comment was added.